db/schema.sql

```diff
@@ -148,17 +202,98 @@ CREATE SEQUENCE result_result_id_seq
     CACHE 1;
 
 
-ALTER TABLE result_result_id_seq OWNER TO gatcrpvp;
+ALTER TABLE result_result_id_seq OWNER TO backquote_user;
 
 --
--- Name: result_result_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: gatcrpvp
+-- Name: result_result_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: backquote_user
 --
 
 ALTER SEQUENCE result_result_id_seq OWNED BY result.id;
 
 
 --
--- Name: trade; Type: TABLE; Schema: public; Owner: gatcrpvp
+-- Name: simulation; Type: TABLE; Schema: public; Owner: backquote_user
+--
+
+CREATE TABLE simulation (
+    id integer NOT NULL,
+    backtest_id integer,
+    ticker character varying(10),
+    params text,
+    profit numeric(10,2)
```

Comment:
Yeah I'll change it to `profit_reset` and `profit_no_reset` since it's for the whole simulation.